backend/src/journal/dto/create-journal.input.ts

```diff
@@ -0,0 +1,13 @@
+import { InputType, Field } from '@nestjs/graphql';
+
+@InputType()
+export class CreateJournalInput {
+  @Field({ nullable: true })
+  id?: string;
```

Comment:
Create Input doesn't need an id — ID is automatically created by the database

Reply:
Done!